webpack/components/Bookmark/index.js

```diff
@@ -4,7 +4,7 @@ import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import { translate as __ } from 'foremanReact/common/I18n';
 import { STATUS } from 'foremanReact/constants';
 import { Dropdown, DropdownItem, DropdownToggle, DropdownSeparator } from '@patternfly/react-core';
-import { CaretDownIcon } from '@patternfly/react-icons';
+import { OutlinedBookmarkIcon } from '@patternfly/react-icons';
```

Comment:
It is not the same icon Foreman uses.

Reply:
@lfu I pulled that one from here:

https://github.com/theforeman/foreman/blob/639b7a366ff1e556d710e24e240186e8e5f4c6f7/webpack/assets/javascripts/react_app/components/PF4/Bookmarks/Bookmarks.js#L4